openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -118,8 +118,10 @@ components:
               Here is a list of some parameters - POSTGRES_SERVER_SKU, POSTGRESQL_SUBNET, VNET_ADDRESS_SPACE.
               These are example of assigned values - POSTGRES_SERVER_SKU=GP_Gen5_2, POSTGRESQL_SUBNET=10.1.0.16/29
             $ref: '#/components/schemas/AzureLandingZoneParameter'
-        landingZoneTarget:
-          $ref: '#/components/schemas/LandingZoneTarget'
+        billingProfileId:

```

Comment:
LZ creation now takes a `billingProfileId` instead of `landingZoneTarget`.

Reply:
Yes, the LZ stores what billing profile it's in.

> Shouldn't there be an API to lookup landing zones from billing profile?

Yes, I think so. In fact the LZ service layer already has a method to lookup by billing profile with the right Sam checks. I can add a REST API too.

> Shouldn't there be an API description in the design doc?

Probably, I can write this down too

Reply:
I wrote down the intended APIs here: https://docs.google.com/document/d/1a0OeD-7hsUXYPMZ8JQ_rjMguHlwbpbUPL9TDkLygHp4/edit#heading=h.5eyy5cpubyu0

I am thinking I will ticket/PR separately adding the missing `list` APIs as it involves some more refactoring in WSM/LZ service.

Reply:
https://broadworkbench.atlassian.net/browse/TOAZ-222